__tests__/app.test.js

```diff
@@ -317,4 +319,39 @@ describe.only("5. PATCH /api/reviews/:review_id", () => {
         expect(body.msg).toBe("Bad request");
       });
   });
+
+  test("should status: 202, even when some other property included on req body (ignore other property", () => {
```

Comment:
Love this test of simply ignoring additional request body properties - and you're actually asserting the 202 here